release notes/v0.41.0.md

```diff
@@ -0,0 +1,64 @@
+
+### `k6 convert` is officially deprecated ([#2714](https://github.com/grafana/k6/pull/2714))
+
+`k6 convert` has been a sub-command that allows users to convert a [HAR file recording](https://en.wikipedia.org/wiki/HAR_(file_format)) of HTTP traffic into a preliminary k6 script that makes roughly the same requests. It has been neglected for a long while and softly deprecated in favor of the newer and more feature-rich [har-to-k6 standalone converter](https://github.com/grafana/har-to-k6).
+
+We have now officially deprecated `k6 convert`. The command still works, and it will continue to do so for a few more k6 versions. However, it's not visible from `k6 --help` and will emit a warning when used. Please see the [documentation for the standalone har-to-k6 converter](https://k6.io/docs/test-authoring/recording-a-session/har-converter/) and [open an issue](https://github.com/grafana/har-to-k6/issues) (or comment in an existing one) if you have any problems with it.
+
+## New Features, enhancements and UX improvements
+
+- [#2679](https://github.com/grafana/k6/pull/2679) added support for `maxReceiveSize` and `maxSendSize` parameters in the [gRPC's `Client.connect()` method](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/). Thanks, @ariasmn!
+- [#2605](https://github.com/grafana/k6/pull/2605) introduced a new `--exclude-env-vars` CLI flag to [`k6 archive`](https://k6.io/docs/misc/archive-command/) that causes it to not include the provided environment variables in the resulting archive bundle's `metadata.json` file.
+- [#2700](https://github.com/grafana/k6/issues/2700) added support for loading gRPC protoset files. Thanks, @jklipp!
+
+## Bug fixes
+
+- [#2678](https://github.com/grafana/k6/pull/2678) fixed the Docker image labels. Thanks, @knittl, for reporting the problem ([#2677](https://github.com/grafana/k6/issues/2677))!
+- [#2689](https://github.com/grafana/k6/pull/2689) fixed the REST API's `Content-Type` response header. Thanks, @wingyplus!
+- [#2691](https://github.com/grafana/k6/pull/2691) fixed the detailed `k6 version` information embedded in the k6 releases.
+- [#2693](https://github.com/grafana/k6/pull/2693) fixed a big in the k6 event loop when a Promise rejection was left unhandled.
+- [#2696](https://github.com/grafana/k6/pull/2696) fixed a problem with HTTP redirects with empty `Location` headers ([#2474](https://github.com/grafana/k6/issues/2474)) by updating the Go version we use to compile k6 to 1.19.x. Thanks, @agilob!
+- [#2705](https://github.com/grafana/k6/pull/2705) fixed a panic in the `k6/net/grpc` module ([#2661](https://github.com/grafana/k6/issues/2661)). Thanks, @c47gao and @robpickerill!
+- [#2738](https://github.com/grafana/k6/pull/2738) fixed a panic when a Promise was rejected with an `undefined` reason.
```

Comment:
```suggestion
- [#2738](https://github.com/grafana/k6/pull/2738) fixed a panic when a Promise was rejected for an `undefined` reason.
```

Reply:
It is "reject with", see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/reject

Reply:
Strictly grammatically speaking, both are possible, but in this context, you're right, my bad 👍🏻